app/api/v1/guardian/guardian.py

```diff
@@ -120,8 +121,10 @@ def create_guardian(
                     status_code=status.HTTP_409_CONFLICT,
                     detail=f"Already exists {data.guardian_id}",
                 )
+    except HTTPException:
+        raise
     except Exception as error:
-        print(sys.exc_info())
+        traceback.print_exc()
```

Comment:
^ cc @SteveMaier-IRT , we discussed lack of stack traces, this is how to do it.  Unfortunately, I couldn't find any way to globally add stack traces.  I'd like to consider implementing project-wide request handling at some point so we could do that kind of thing, but it's way beyond the scope of this PR.